tests/common/dualtor/dual_tor_utils.py

```diff
@@ -185,19 +185,31 @@ def update_mux_configs_and_config_reload(dut, state):
     dut.file(path=TMP_FILE, state='absent')
 
 
-def force_active_tor(dut, intf):
+@pytest.fixture
+def force_active_tor():
     """
     @summary: Manually set dut host to the active tor for intf
     @param dut: The duthost for which to toggle mux
     @param intf: One or a list of names of interface or 'all' for all interfaces
     """
-    if type(intf) == str:
-        cmds = ["config muxcable mode active {}".format(intf)]
-    else:
-        cmds = []
-        for i in intf:
-            cmds.append("config muxcable mode active {}".format(i))
-    dut.shell_cmds(cmds=cmds)
+    forced_intfs = []
+    def _force_active_tor(dut, intf):
+        if type(intf) == str:
+            cmds = ["config muxcable mode active {}; true".format(intf)]
+            forced_intfs.append((dut, intf))
+        else:
+            cmds = []
+            for i in intf:
+                forced_intfs.append((dut, i))
+                cmds.append("config muxcable mode active {}; true".format(i))
+        dut.shell_cmds(cmds=cmds, continue_on_fail=True)
+
+    yield _force_active_tor
+
+    if bool(forced_intfs):
```

Comment:
This bool check may not be needed as the below loop would be iterated only if `forced_intfs` list contains something.

Reply:
True, will remove